index.ts

```diff
@@ -38,6 +47,18 @@ function* yamlEntries(root: string): Generator<[string, any]> {
     for (const fp of filePaths) {
         // The feature identifier/key is the filename without extension.
         const { name: key } = path.parse(fp);
+        const pathParts = fp.split(path.sep);
+
+        // Assert ID uniqueness
+        for (const [pool, map] of Object.entries(uniqueIdMaps)) {
+            if (!pathParts.includes("spec") && pathParts.includes(pool)) {
```

Comment:
Is the `!pathParts.includes("spec")` condition about the features/draft/spec/ directory? If so, should we instead give all drafts the same treatment? We set `data[draft]` in this loop so we might be able to use that in the condition.